client/src/components/list/List.tsx

```diff
@@ -0,0 +1,157 @@
+import 'ag-grid-enterprise';
+import { ColDef, GridOptions } from "ag-grid-enterprise";
+import { AgGridReact } from "ag-grid-react";
+import "ag-grid-community/styles/ag-grid.css";
+import "ag-grid-community/styles/ag-theme-quartz.css";
+import { useCallback, useMemo, useRef, useState } from "react";
+import { KxDocument } from "../../model";
+import locales from "../../locales.json";
+import { Button, Flex } from '@tremor/react';
+import { useNavigate } from 'react-router-dom';
+import { RiDeleteBinLine, RiEditLine, RiInfoI } from '@remixicon/react';
+import API from '../../API';
+import { toast } from '../../utils/toaster';
+import { ObjectId } from 'mongodb';
+interface ListProps {
+    documents: KxDocument[];
+}
+
+function List(props: ListProps) {
+    const navigator = useNavigate();
+    const gridRef = useRef<AgGridReact<KxDocument>>(null);
+    const onFirstDataRendered = useCallback(() => {
+        onGridReady();
+    }, []);
+    const infoButton = (rowNode: any) => {
+        console.log(rowNode)
+        return (
+            <Flex justifyContent="evenly" className='mt-1'>
+                <Button size="xs" icon={RiInfoI} onClick={() => navigator('/documents/' + rowNode.value)} />
+                <Button size="xs" icon={RiDeleteBinLine} onClick={async () => {
+                    try {
```

Comment:
I think that it's better to not delete the document after the click on the button to prevent user mistakes.
We can open a dialog in which we ask to the user if he/she is sure and after that, maybe delete the document.